src/vm/hints/blake2s_utils.rs

```diff
@@ -71,19 +71,19 @@ fn compute_blake2s_func(
    compute_blake2s_func(segments=segments, output_ptr=ids.output)
 */
 pub fn compute_blake2s(
-    vm: &mut VirtualMachine,
+    variables: HintVisibleVariables,
     ids: &HashMap<String, BigInt>,
     hint_ap_tracking: Option<&ApTracking>,
 ) -> Result<(), VirtualMachineError> {
     let output = get_ptr_from_var_name(
         "output",
         ids,
-        &vm.memory,
-        &vm.references,
-        &vm.run_context,
+        variables.memory,
+        variables.references,
+        variables.run_context,
         hint_ap_tracking,
```

Comment:
This could receive the `variables` object directly, right?

Reply:
Done!